server/services/statementService.test.js

```diff
@@ -43,6 +44,8 @@ describe('getStatement', () => {
     expect(output).toEqual(incident)
 
     expect(incidentClient.getStatement).toBeCalledWith('BOB', 1, StatementStatus.PENDING)
+
+    expect(incidentClient.getAdditionalComments).toBeCalled()
```

Comment:
should probably test the merging too: 

* rename `incident` -> `statement`
* add `incidentClient. getAdditionalComments.mockReturnValue([... one or more comments])`
* update `expect(output).toEqual(incident)` to `expect(output).toEqual({...statement, additionalComments: [...])`

Reply:
done